results/gtkCustomTheme.md

```diff
@@ -0,0 +1,49 @@
+# Custom theming support by GTK
+
+GTK (GIMP Toolkit) provides robust support for theming, allowing users and developers to customize the appearance of GTK-based applications. 
+
+**GTK provides a theme engine** that renders widgets based on the theme selected. Themes are usually defined using CSS (Cascading Style Sheets), where colors, borders, sizes, fonts, and other visual properties are defined.
+
+**Theme Files:** Custom GTK themes are typically stored in directories like `~/.themes` (Linux) or system-wide locations such as `/usr/share/themes`.
+On Windows it is located in `C:\msys64\mingw64\share\themes` when Msys2 (mingw64) is used.
+
+**Theme format:** GTK+ 3.x and GTK+ 4.x - These modern versions use CSS for styling. Style rules are defined in gtk.css files, allowing much more flexibility.
+
+**GTK Theme Application:** 
+To apply a custom theme in GTK, use the `GTK_THEME` environment variable or configure the settings in the `~/.config/gtk-3.0/settings.ini` file.
+GTK automatically detects themes placed in specific directories like `~/.themes` or `/usr/share/themes` or `C:\msys64\mingw64\share\themes` (on Windows with Msys2 and mingw64).
+
+**Eclipse SWT and GTK Custom themes**
+
+  - **SWT on Linux :**
+     +  Download custom GTK3/GTK4 themes. (I did it from https://www.pling.com/) . Place it under `/usr/share/themes` location and unzip.
+
+         ![image](https://github.com/user-attachments/assets/64d97283-6960-446a-a493-16d3e2e97a1d)
+
+     +   Add `GTK_THEME` environment variable with the value being the name of the themes folder (Example : Dracula) in the run configuration of GTK application(Eclipse SWT Application or Control example or Snippets).
+
+          ![image](https://github.com/user-attachments/assets/bf8190c4-2585-490e-b3af-6c16ada6b5ea)
```

Comment:
Is the application **GTK_ON_WINDOWS** also relevant? 

I think not. If I'm right then please blur it and let only **SWT_ON_WINDOWS** be visible.